exercises/basic_reduce/solution/solution.js

```diff
@@ -1,8 +1,8 @@
 function countWords(arr) {
-  return arr.reduce(function(countMap, word) {
-    countMap[word] = ++countMap[word] || 1 // increment or initialize to 1
-    return countMap
-  }, {}) // second argument to reduce initialises countMap to {}
+  return arr.reduce(function(accumulator, current_word) {
```

Comment:
I see your point.  Thinking of it as an accumulator helped me understand reduce better, but using semantic naming is also helpful.